functions/packages.sh

```diff
@@ -40,6 +40,50 @@ firemotd_setup() {
   fi
 }
 
+create_mta_config() {
+  TMP=/tmp/.$$
+  HOSTNAME=`/bin/hostname`
+  INTERFACES="`/usr/bin/dig +short $HOSTNAME | /usr/bin/tr '\n' ';'`;127.0.0.1;::1"
+  EXIM_CLIENTCONFIG=/etc/exim4/passwd.client
+
+  introtext1="We will guide you through the install of exim4 as the mail transfer agent on your system and configure it to relay mails through a public service such as Google gmail.\nPlease
+enter the data shown in the next window when being asked for. You will be able to repeat the whole installation if required by selecting the openHABian menu for MTA again."
+  introtext2="Mail server type: mail sent by smarthost; received via SMTP or fetchmail\nSystem mail name: FQDN (your full hostname including the domain art)A\nIPs should be allowed by the se
```

Comment:
Sorry but I don't understand this text. I don't get what will follow if I accept this. What is an `smarthost` for example? Remember the problem openhabian tries to solve: **A home automation enthusiast doesn't have to be a Linux enthusiast!**

If you need to include application specific terms and concept refer the reader to relevant documentation in the dialog box.

Reply:
Check text in general (newline, lowercase, uppercase etc.)

Reply:
I believe you misunderstood this. Actually the exim install routine - once called - will prompt the user with these texts. Now we cannot change the exim package to do that, but we can tell the user what he'll have to enter. That's what my window here is telling the user: what to enter if he's (later on) asked for a selection. He doesn't need to understand what a smarthost is just to make that choice.

  introtext2="Mail server type: mail sent by smarthost; received via SMTP or fetchmail\nSystem mail name: FQDN (your full hostname including the domain art)A\nIPs should be allowed by the se
```

Reply:
@EliasGabrielsson now that benjy had his look, could you also have have another one please?